api/src/http.rs

```diff
@@ -31,6 +31,8 @@ pub struct ApiMountCmd {
     pub fs_type: String,
     /// Configuration for the filesystem.
     pub config: String,
+    /// Authorization for private registry.
+    pub auth: Option<String>,
```

Comment:
Why is this extension needed? I think callers should merge the auth info into `config`.

Reply:
Yes, you are correct. Fixed it now.